src/code.cloudfoundry.org/policy-server/store/migrations/v0067.go

```diff
@@ -0,0 +1,71 @@
+package migrations
+
+// These all use different migrations vs multiple statements in a text block,
+// or multiple text blocks per migration, because of errors encountered
+// with the migrator not running migrations in a transaction in mysql properly
+// See commit ID f1280ed0eee131aa57bd9af0cdbc590fb36be80a for more information
+
+var migration_v0067a = map[string][]string{
+	"mysql": []string{
+		`CREATE TABLE IF NOT EXISTS "security_groups" (
```

Comment:
let's pull the change on our branch that adds id and uses staging_group_spaces and running_group_spaces as JSON fields before merging this migration